model/adjuster/span_merger.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2018 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adjuster
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// MergeSpans returns an Adjuster that merges Jaeger spans with the same spanID.
+// Duplicate spans that have conflicting span.kind annotations are not merged.
+//
+// MergeSpans assumes that the duration field in a span is monotonically increasing for a spans with the
+// same spanID. The span with the longest spanID wins, and is selected in entirety.
```

Comment:
s/spanID/duration

Reply:
This is a change from Pull Request #728 

Shouldn't it be changed there then?